exercise-1.js

```diff
@@ -119,36 +180,52 @@
             // console.log(" the value of i in the loop is : " + i);
         }
 
+        let i = 0;
+        while (i < 10 ) {
+            console.log(" the value of i in the loop is : " + i);
+            i++
 
-        //your code...
-
-
+        }
 
+        //your code...
 
         /************************************************************* */
         //Multiply the sum of 30 added to two by 20.  Divide the product by 10 raised to the power of 2
         //use javascript to compute the value of the above statement. Each individual operation needs to be a function expression. run all the functions after defining them, and print the answer to the console.
 
         //your code...
-
+        console.log("Amir's Equation: " + " ((30 + 2) * 20) / (10 ** 2) = " + (((30 + 2) * 20) / (10 ** 2)))
```

Comment:
I liked that you did it all in one line, but the exercise is asking to create a function for each individual operation.